README/INSTALL

```diff
@@ -7,81 +7,49 @@ Contents:
+   3.2.  Fixed location build
    4.    Installation Location
-   5.    Makefile targets
-   6.    Installing optional add-on libraries
-   7.    Information on third party software
-   7.1.  Posix Thread usage
-   7.2.  OpenGL library
-   7.3.  SHIFT managed tape I/O
-   7.4.  ZEBRA file conversion tools
-   7.5.  MySQL client
-   7.6.  PostGreSQL Client
-   7.7.  SapDB Client
-   7.8.  Pythia Event Generators
-   7.9.  Venus Event Generators
-   7.10. Secure Remote Password (SRP) Authentication
-   7.11. AFS Authentication
-   7.12. Kerberos Authentication
-   7.13. Globus Authentication
-   7.14. SSH Authentication
+   5.    Where to get more information
+   6.    Where to get help
 
 1. Getting the source:
 ======================
 
-To install ROOT from source you first have to get the tar file
-containing the source. This tar file can be found in the usual
-ROOT download page http://root.cern.ch/downloading-root. The files are named
-root-<version>.source.tar.gz.
+To install ROOT from source you first have to get the source code.
 
-Here's a short summary:
+You can clone the source code from our GitHub repository:
 
-
-1) Get the sources
-
-     wget https://root.cern.ch/download/root_<version>.source.tar.gz
-
-2) Unpack the distribution:
-
-     tar -zxf root_<version>.source.tar.gz
-
-An alternative approach is to use our public GIT repository to get
-the latest version.
-
-     git clone http://root.cern.ch/git/root.git
+     git clone https://github.com/root-project/root.git
 
 The release specific tag can be obtained using for example:
 
      cd root
-     git checkout -b v6-04-14 v6-04-14
-
+     git checkout v6-22-00
 
 
 2. Getting ready to build:
 ==========================
 
-Building ROOT requires to have CMake in your system. This can be
-downloaded from http://cmake.org
-You may want to compile features into ROOT, which depends on third
-party libraries. Make sure you meet all demands for additional
+ROOT requires CMake to be built.
+You may want to compile features into ROOT, which depend on third-party libraries.
+Make sure you meet all demands for additional
 features before trying to enable them (see below).
 
-The prerequisites needed for the different platforms are documented at 
+The prerequisites needed for the different platforms are documented at
 
-  https://root.cern.ch/build-prerequisites
+  https://root.cern/install/dependencies
 
 
 3. Choosing the installation method:
 ====================================
 
 There are two main methods of installing ROOT from source.
   - 'location independent' is advised for a personal installation of ROOT
-  - 'fix location' for a system wide installation. 
-  
-Both require to set the CMAKE_INSTALL_PREFIX variable at configuration time. 
-The 'fix location' requires in addition to set the 'gnuinstall' option at configuration. 
+  - 'fixed location' for a system wide installation.
+
+Both require to set the CMAKE_INSTALL_PREFIX variable at configuration time.
```

Comment:
Is that really true? I +/- never use it as I +/- never `make install`.

Reply:
I guess they don't _require_ it, they both _honor_ it (and the default is `/usr/local`). I'll clarify.